src/proj_internal.h

```diff
@@ -57,6 +57,9 @@ extern "C" {
 #define PJ_TORAD(deg)  ((deg)*M_PI/180.0)
 #endif
 
+/* Maximum latitudinal overshoot accepted */
+#define PJ_EPS_LAT 1e-12
```

Comment:
While there are many cases of EPS, they are not all the same. This is a first step towards having a centralized definition of "the EPS that is used when checking for latitudinal overshoot". The other EPSes should also be given more meaningful names, but in some cases it is not at all evident what that should be, because the intention is unclear. So writing up a meaningful ticket on this is not a small task, although I agree it should be done.

Reply:
> While there are many cases of EPS, they are not all the same

Yes. Many are used in iterative inverse projections. Others are used to check for latitudinal overshoot. I think both of these cases should be fairly easy to spot and differentiate. There will likely be a handful of EPS's left of course.